helix-view/src/theme.rs

```diff
@@ -164,6 +171,39 @@ fn parse_color(value: Value) -> Option<Color> {
     }
 }
 
+fn parse_modifier(value: Value) -> Option<Modifier> {
+    if let Value::String(s) = value {
+        match s.as_str() {
+            "bold" => Some(Modifier::BOLD),
+            "dim" => Some(Modifier::DIM),
+            "italic" => Some(Modifier::ITALIC),
+            "underlined" => Some(Modifier::UNDERLINED),
+            "slow_blink" => Some(Modifier::SLOW_BLINK),
+            "rapid_blink" => Some(Modifier::RAPID_BLINK),
+            "reversed" => Some(Modifier::REVERSED),
+            "hidden" => Some(Modifier::HIDDEN),
+            "crossed_out" => Some(Modifier::CROSSED_OUT),
+            _ => None,
+        }
+    } else {
+        None
+    }
+}
+
+fn parse_modifiers(value: Value) -> Option<Vec<Modifier>> {
+    if let Value::Array(arr) = value {
+        let mut modifiers = Vec::new();
+        for val in arr {
```

Comment:
I think this could be more concise with filter_map:

     let modifiers = arr.iter().filter_map(parse_modifier)